image/vulnerabilities/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8-minimal
+ARG BASE_TAG=latest
+
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
+
+ARG LABEL_VERSION
+ARG LABEL_RELEASE
+ARG QUAY_TAG_EXPIRATION
+
+LABEL name="definitions" \
+      vendor="StackRox" \
+      maintainer="https://stackrox.io/" \
+      summary="Vulnerability definitions for the StackRox Security Platform" \
+      description="This image contains vulnerability definitions used by StackRox Scanner V4." \
```

Comment:
you can just say `StackRox scanners` since it's for both

Reply:
Good catch. That was from when I thought this was only related to Scanner V4 😅